backend/app/utils/Ocr.scala

```diff
@@ -47,9 +47,9 @@ object Ocr {
 
   // TODO MRB: allow OcrMyPdf to read DPI if set in metadata
   // OCRmyPDF is a wrapper for Tesseract that we use to overlay the OCR as a text layer in the resulting PDF
-  def invokeOcrMyPdf(lang: String, inputFileName: String, dpi: Option[Int], stderr: mutable.Buffer[String], tmpDir: Path): Path = {
-    val tempFile = Files.createTempFile("ocrmypdf", ".pdf")
-    val cmd = s"ocrmypdf --redo-ocr -l $lang ${dpi.map(dpi => s"--image-dpi $dpi").getOrElse("")} ${inputFileName} ${tempFile.toAbsolutePath}"
+  def invokeOcrMyPdf(lang: String, inputFilePath: Path, dpi: Option[Int], stderr: mutable.Buffer[String], tmpDir: Path): Path = {
```

Comment:
would it be any clearer/less error-prone to just have one param representing full path to file?

Reply:
Ah do you mean for the temporary file? I'm not sure it would be as we'd repeat the same logic in all the calling extractors